### PR TITLE
doc: Fix typos

### PR DESCRIPTION
**Pull request by georgeto**

### CODE REVIEW

File: doc/tutorial/compile.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Build a demo project
 ********************
 
-Prequisites
+Prerequisites
 ===========
```
Comment (jkloetzke) [inline doc/tutorial/compile.rst:5]: I'm afraid rst requires that the underlining must be at least as long as the text...

```suggestion
=============
```

**Review by jkloetzke — CHANGES_REQUESTED** [review]
Thanks for fixing the typos.

File: doc/tutorial/compile.rst (hunk 1 above)
Comment (georgeto) [reply doc/tutorial/compile.rst]: You never stop learning ;)

**Comment by jkloetzke** [comment]
Thanks!

**Pull request merged**